bismuth.spec

```
@@ -29,19 +30,67 @@ them via keyboard, just like in classical tiling window managers
+%{_datadir}/kpackage/kcms/kcm_bismuth
+%{_datadir}/kpackage/kcms/kcm_bismuth/metadata.desktop
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/views
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/views/WorkspaceRules.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/views/WindowRules.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/views/MonocleOverlay.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/views/Layouts.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/main.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/components
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/components/RatioConfigSpinBox.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/components/PixelsConfigSpinBox.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/components/ConfigTextField.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/components/ConfigSpinBox.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/components/ConfigCheckBox.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/Behaviour.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/contents/ui/Appearance.qml
+%{_datadir}/kpackage/kcms/kcm_bismuth/metadata.json
+%{_datadir}/icons/hicolor/scalable/apps/bismuth.svg
 %{_datadir}/icons/hicolor/22x22/categories/bismuth-kcm.svg
 %{_datadir}/icons/hicolor/64x64/categories/bismuth-kcm.svg
-%{_datadir}/icons/hicolor/scalable/apps/bismuth.svg
-%{_datadir}/kconf_update/old_conf_ui.sh
-%{_datadir}/kconf_update/old_conf_ui.upd
-%{_datadir}/kpackage/kcms/kcm_bismuth
-%{_datadir}/kservices5/kcm_bismuth.desktop
 %{_datadir}/kwin/scripts/bismuth
+%{_datadir}/kwin/scripts/bismuth/contents
+%{_datadir}/kwin/scripts/bismuth/contents/ui
+%{_datadir}/kwin/scripts/bismuth/contents/ui/main.qml
+%{_datadir}/kwin/scripts/bismuth/contents/ui/popup.qml
+%{_datadir}/kwin/scripts/bismuth/contents/code
+%{_datadir}/kwin/scripts/bismuth/contents/code/index.mjs
+%{_datadir}/kwin/scripts/bismuth/metadata.desktop
+%{_datadir}/icons/hicolor/16x16/status/bismuth-column.svg
+%{_datadir}/icons/hicolor/16x16/status/bismuth-floating.svg
+%{_datadir}/icons/hicolor/16x16/status/bismuth-monocle.svg
+%{_datadir}/icons/hicolor/16x16/status/bismuth-quarter.svg
+%{_datadir}/icons/hicolor/16x16/status/bismuth-spiral.svg
+%{_datadir}/icons/hicolor/16x16/status/bismuth-spread.svg
+%{_datadir}/icons/hicolor/16x16/status/bismuth-stair.svg
+%{_datadir}/icons/hicolor/16x16/status/bismuth-tile.svg
+%{_datadir}/icons/hicolor/32x32/status/bismuth-column.svg
+%{_datadir}/icons/hicolor/32x32/status/bismuth-floating.svg
+%{_datadir}/icons/hicolor/32x32/status/bismuth-monocle.svg
+%{_datadir}/icons/hicolor/32x32/status/bismuth-quarter.svg
+%{_datadir}/icons/hicolor/32x32/status/bismuth-spiral.svg
+%{_datadir}/icons/hicolor/32x32/status/bismuth-spread.svg
+%{_datadir}/icons/hicolor/32x32/status/bismuth-stair.svg
+%{_datadir}/icons/hicolor/32x32/status/bismuth-tile.svg
```

You can use wildcards here
```suggestion
%{_datadir}/icons/hicolor/16x16/status/bismuth*
%{_datadir}/icons/hicolor/32x32/status/bismuth*
```